totalRP3/UI/Widgets.lua

```diff
@@ -201,3 +203,82 @@ end
 function TRP3_API.GetLastCopiedIcon()
 	return g_lastCopiedIcon;
 end
+
+local g_lastCopiedColor;
+
+--- TRP3_API.SetLastCopiedColor sets the last copied color.
+---@param color string|table Contains the color to be copied.
+function TRP3_API.SetLastCopiedColor(color)
+	g_lastCopiedColor = color;
+end
+
+--- TRP3_API.GetLastCopiedColor gets the last copied color.
+---@return string|table color Contains the last color that was copied.
+function TRP3_API.GetLastCopiedColor()
+	return g_lastCopiedColor;
+end
+
+TRP3_ColorPickerButtonMixin = {};
+
+function TRP3_ColorPickerButtonMixin:OnLoad()
+	TRP3_ColorButtonLoad(self);
+end
```

Comment:
Could the body of TRP3_ColorButtonLoad be inlined into this OnLoad function, and TRP3_ColorButtonLoad removed?

Not sure if Extended is using it.

Reply:
Doesn't seem like it does, so it should be fine